data.sql

```diff
@@ -10,4 +10,26 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Angemon', '06-12-2005', 1, true, -45);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Boarmon', '06-07-2005', 7, true, 20.4);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Blossom', '10-13-1998', 3, true, 17);
-INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Ditto', '05-14-2022', 4, true, 22);
\ No newline at end of file
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Ditto', '05-14-2022', 4, true, 22);
+
+-- inserting data in "owners" table
+INSERT INTO owners (full_name, age) VALUES('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES('Jodie Whittaker', 38);
+
+-- inserting data in "species" table
+INSERT INTO species (name) VALUES('Pokemon');
+INSERT INTO species (name) VALUES('Digimon');
+
+-- updating data in "animals" table
+UPDATE animals SET species_id=2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id=1 WHERE species_id IS NULL;
```

Comment:
- [ ] Dear @Fombi-Favour you did a great job, and your query is working fine. But I think your approach is not correct, you are trying to update things by adding the id statically, just think about a scenario, where you need to make the changes but you don't have any idea about the `ID` of the species. Kindly look at the below hint to understand to update the column data more efficiently.

## Hint
![image](https://github.com/Fombi-Favour/vet-clinic-sql/assets/66476329/408a13bf-f04a-4572-83b6-1924818475a3)